web/sites/default/config/views.view.media.yml

```diff
@@ -3920,16 +3920,7 @@ display:
     position: 1
     display_options:
       display_extenders: {  }
-      path: admin/content/media
-      menu:
-        type: tab
-        title: Media
-        description: ''
-        expanded: false
-        parent: ''
-        weight: 0
-        context: '0'
-        menu_name: main
+      path: admin/content/media-table
```

Comment:
Not sure if this path was supposed to change or not (and both appear to still work). I also just noticed there's two "media" tabs now on the /admin/content page.